### PR TITLE
nerfs hlin to acceptable armor levels

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
95 to 25, so it isn't full immunity, more like 75% with all armor pieces 

## Why It's Good For The Game
full immunity was a mistake

## Changelog
:cl:
balance: hlin no longer gives you full bomb immunity, only around 80%
/:cl:

### CODE REVIEW

**Comment by SplinterGP** [comment]
Give better reason

**Comment by Pariah919** [comment]
>Give better reason
>
hlin giving you full immunity isn't a good thing because it allows you to chian pointblank grenades at xenomorphs who have difficulty escaping that scenario, immunity to OBs (Which is insanely silly), immunity to getting CASed, immunity to Claymores and other misc explosions
This makes it chip damage you when you get grenaded with light explosoions, which are the most common overall explosions so I think it's fine.
It's way too strong for 12 points, Near immunity is better for the game.

**Comment by DerrickChamberlain** [comment]
Hiln mod on suicide watch.

**Comment by SkeletalElite** [comment]
hlin is garbage though

**Comment by InterroLouis** [comment]
Killing the Hlin's only purpose, my god.

File: code/modules/modular_armor/attachables.dm [hunk 1]
```diff
@@ -174,7 +174,7 @@
 	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Uses a complex set of armor plating and compensation to lessen the effect of explosions, at the cost of making the user slower."
 	icon_state = "mod_boomimmune_icon"
 	item_state = "mod_boomimmune"
-	soft_armor = list("bomb" = 95)
+	soft_armor = list("bomb" = 25)
```
Comment (Pariah919) [inline code/modules/modular_armor/attachables.dm:177]:
```suggestion
	soft_armor = list("bomb" = 30)
```

**Comment by David-Stormwell** [comment]
Reasonable change now being reduced to 80%, I'd ask for a TMC but knowing Maintainers they'd rather close or merge this.

Also reminder for marine mains that shields, pre-med and other things exist to soften the damage.

**Comment by InterroLouis** [comment]
> Reasonable change now being reduced to 80%, I'd ask for a TMC but knowing Maintainers they'd rather close or merge this.
> 
> Also reminder for marine mains that shields, pre-med and other things exist to soften the damage.
> Additionally B17 suit exists.

If the latest shield nerf PR gets in then shields will not in fact, soften the damage from grenades at all.

**Comment by Pariah919** [comment]
>If the latest shield nerf PR gets in then shields will not in fact, soften the damage from grenades at all.
>
nope, go read it again

**Comment by SplinterGP** [comment]
> 
> 
> Reasonable change now being reduced to 80%, I'd ask for a TMC but knowing Maintainers they'd rather close or merge this.
> 
> Also reminder for marine mains that shields, pre-med and other things exist to soften the damage.
> Additionally B17 suit exists.

B17 exists for giving 70 armor and full bomb protection and 0.55 slowdown, not only give bomb protection, Hlin gives 0.2 slowdown in exchange of full bomb protection, which xenos cant cause, thats only self damage

**Comment by SpaceLoveSs13** [comment]
Hlin should be nerfed but not to this value. On another hand maybe buff B17 so people get interested into buying the actual bomb protective armor as a reward to getting kills and points instead of a 12 points worth module.

**Comment by SplinterGP** [comment]
Hlin doens't deserve this nerf, it's only usefull against SELF damage as basically none of the xenos cause explosion damage, most usefull if youre running a explosives loadout which then is very much especialized and has its downsides such as grenade scarcity, the module also gives slowdown and youre paying for it, not for free.

**Pull request closed without merging**